src/button.tsx

```diff
@@ -475,10 +476,14 @@ interface ButtonLinkToProps extends ButtonLinkCommonProps {
 
 export type ButtonLinkProps = ButtonLinkOnPressProps | ButtonLinkHrefProps | ButtonLinkToProps;
 
-export const ButtonLink = React.forwardRef<TouchableElement, ButtonLinkProps>((props, ref) => {
+export const BaseButtonLink = React.forwardRef<
+    TouchableElement,
+    ButtonLinkProps & {type: ButtonLinkType; withChevron?: boolean}
+>(({type, ...props}, ref) => {
     const {formStatus} = useForm();
     const isInverse = useIsInverseVariant();
-    const {analytics, textPresets} = useTheme();
+    const {textPresets} = useTheme();
+    const {eventFormat} = useTrackingConfig();
```

Comment:
Unrelated change: `eventFormat` should be imported by `useTrackingConfig()`, not from `useTheme()`

Reply:
remember to close the related jira ticket with this bug